<!-- -->
<launch>
	<arg name="file"/>
	<node pkg="pcd_tools" type="pcdfilter_node" name="pcdfilter_node" output="screen">
		<param name="pcd_file" value="$(arg file)"/>
		<param name="clip_range_up" value="-0.1"/>
		<param name="clip_range_down" value="-1.3"/>
		
	</node>
	
</launch>

